hudi-common/src/main/java/org/apache/hudi/common/table/timeline/TimelineUtils.java

```diff
@@ -343,7 +343,8 @@ public static HoodieTimeline handleHollowCommitIfNeeded(HoodieTimeline completed
     switch (handlingMode) {
       case FAIL:
         throw new HoodieException(String.format(
-            "Found hollow commit: '%s'. Adjust config `%s` accordingly if to avoid throwing this exception.",
+            "Found hollow commit: '%s'. If this happened for an incremental query, "
```

Comment:
lets also add time travel query in addition to incremental query.

Reply:
this method is now only used for incr query. added a dedicated one for time travel. so no change needed here.